lib/services/devices/deviceService.js

```diff
@@ -234,7 +234,7 @@ function mergeArrays(original, newArray) {
         newKeys = _.pluck(newArray, 'object_id'),
```

Comment:
An entry in CHANGES_NEXT_RELEASE file should be added describing what this PR is fixing (have a look to existing content to CHANGES_NEXT_RELEASE entries to see what I'm referring to).

Reply:
If this PR is solving more than one issue, then one entry per issue should be added.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/551/commits/3753c201c702080240ab07947392e895ca7519c4